httpdebug/context.go

```diff
@@ -0,0 +1,15 @@
+package httpdebug
+
+import (
+	"context"
+)
+
+type ctxKey struct{}
+
+func isDebugModeEnabled(ctx context.Context) bool {
```

Comment:
Keep it exported, so we can use it anywhere we want.
```suggestion
func IsDebugModeEnabled(ctx context.Context) bool {
```